Utilities/src/main/java/edu/stanford/hivdb/mutations/Apobec.java

```diff
@@ -25,7 +24,7 @@
 import edu.stanford.hivdb.utilities.Cachable;
 
 public class Apobec {
-
+		
```

Comment:
Empty spaces here and there. Just remove them in the future.

Reply:
Will do. Also going to look into helpful plugins to minimize these hiccups (For example, I believe AnyEdit has a feature that removes trailing white space on save).